test/plugin/test_parser_syslog.rb

```diff
@@ -133,6 +150,21 @@ def test_parse_with_rfc5424_message_without_time_format
       end
     end
 
+    def test_parse_with_rfc5424_message_with_priority_and_pid
+      @parser.configure(
+                        'message_format' => 'rfc5424',
+                        'with_priority' => true,
+                        )
+      text = '<28>1 2018-09-26T15:54:26.620412+09:00 machine minissdpd 1298 - -  peer 192.168.0.5:50123 is not from a LAN'
+      @parser.instance.parse(text) do |time, record|
+        assert_equal(event_time("2018-09-26T15:54:26.620412+0900", format: '%Y-%m-%dT%H:%M:%S.%L%z'), time)
```

Comment:
Time format should be %Y-%m-%dT%H:%M:%S.**%6N**%z instead of %Y-%m-%dT%H:%M:%S.**%L**%z.

Reply:
Umm, this format seems to be correct... :confounded: